pkg/virt-config/configuration_test.go

```diff
@@ -424,8 +424,18 @@ var _ = Describe("test configuration", func() {
 		Entry("when unset, GetSELinuxLauncherType should return the default", virtconfig.DefaultSELinuxLauncherType, virtconfig.DefaultSELinuxLauncherType),
 	)
 
-	DescribeTable(" when OVMFPath", func(cpuArch string, ovmfPathKeyAMD64 string, ovmfPathKeyARM64 string, ovmfPathKeyPPC64le64 string, result string) {
+	DescribeTable(" when AppArmorLauncherProfile", func(value string, result string) {
+		clusterConfig, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{
+			AppArmorLauncherProfile: value,
+		})
+		appArmorProfile := clusterConfig.GetAppArmorLauncherProfile()
+		Expect(appArmorProfile).To(Equal(result))
+	},
+		Entry("when set, GetAppArmorLauncherProfile should return the value", "defaultAAprofile", "defaultAAprofile"),
+		Entry("when unset, GetAppArmorLauncherProfile should return the default", virtconfig.DefaultAppArmorLauncherProfile, virtconfig.DefaultAppArmorLauncherProfile),
+	)
 
+	DescribeTable(" when OVMFPath", func(cpuArch string, ovmfPathKeyAMD64 string, ovmfPathKeyARM64 string, ovmfPathKeyPPC64le64 string, result string) {
```

Comment:
As mention above you can omit all the `string` and only leave the type on the last argument